app/next.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
   publicRuntimeConfig: {
     NO_MATHJAX: process.env.NO_MATHJAX,
     FEEDBACK_SITE_URL: process.env.FEEDBACK_SITE_URL,
+    SUPPORT_EMAIL: process.env.SUPPORT_EMAIL,
```

Comment:
Should `FEEDBACK_SITE_URL` be deleted (and all of its occurences replaced by `SUPPORT_EMAIL`)?

Reply:
I had wondered that but left it as-is, since we intend to replace fider with a new solution that is distinct from a support email.

Reply:
Perhaps the best compromise in the meantime (until we have implemented that solution) would be to keep the FEEDBACK_SITE_URL variable, but temporarily replace all usages in the app with SUPPORT_EMAIL.